### DISCUSSION TITLE
cookies problem

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5552] [Help]
hi everyone please help me with this issue 

i have 2 language (English & Arabic)
so when the project load first in 1 language 
then a cookie saved as localhost:35262/
then if i changed to another language and i am not on root as example if i am on localhost/adminstration/language 
then i changed the language to arabic
then a new cookie will saved as   localhost/adminstration/ 
with arabic content 
so when i press on links in that path it will always convert me to arabic language even if in English mood

how can i solved this problem 
plese help me it is urgent

**Comment by VictorTomaili** [comment]
Please continue from #5549 
Don't spam with same issue two or more times.
